lib/rules/comma-dangle.js

```diff
@@ -144,23 +143,33 @@ module.exports = {
          * @returns {ASTNode|null} The last node or null.
          */
         function getLastItem(node) {
+
+            /**
+             * Returns the last element of an array
+             * @param {any[]} array The input array
+             * @returns {any} The last element
+             */
+            function last(array) {
+                return array[array.length - 1];
+            }
+
```

Comment:
there is a stage-3 proposal: https://github.com/tc39/proposal-relative-indexing-method
so it will be written to `array.at(-1)` someday. 😄